scripts/cob-command

```diff
@@ -194,9 +195,15 @@ start_session(){
     echo -e "${red}ERROR: $arg_session_name session already exist${NC}"
     echo -e "${black} ${NC}"
   else
+    if ! [ -z $arg_pre_condition ]; then
+      arg_command=$arg_pre_condition' && '$arg_command
+    fi
     send_command=${arg_command//" "/"\ "}
-    su $arg_user -l -c "ssh $arg_pc 'tmux new -d -s $arg_session_name'"
-    su $arg_user -l -c "ssh $arg_pc 'tmux send -t $arg_session_name $send_command ENTER'"
+    send_command=${send_command//"&"/"\&"}
```

Comment:
tmux has problems with some characters , theoretically setting the LOCALE variables and using the argument -u (tmux -u) should solve the problem, but this was not working for me.. ideas are welcomed